docs/src/docs/use-cases.md

```diff
@@ -1,121 +1,86 @@
 
-When the second [integration pipeline that Nida kicked off](#ai-ml-and-app-team-collaboration) completed successfully it automatically notified Annika in the SRE team. Annika quickly looks over the integration test results and sees that the model and application are ready for deployment. She issues a PR to Weyland-Yutani's GitOps repository to kick off the production deployment pipeline in GitLab.
+If the model passed validation the QA team would retag it from `tuned` to `challenger` to alert the SRE team that it should be readied for deployment.
 
-Part of the GitOps automation calls the Kit CLI to extract only the model (saving a heap of time by not having to pull a +10GB dataset that isn't needed):
+```shell
+kit tag registry.gitlab.com/chatbot/legalchat:tuned registry.gitlab.com/chatbot/legalchat:challenger
 
-```sh
-$ ./kit pull -filter model corp-registry/app-model:challenger
+kit push registry.gitlab.com/chatbot/legalchat:challenger
 ```
 
-The deployment pipeline does an A/B deployment of both the model and application to their Kubernetes environment in one of their smaller regions (using Kserve). Annika is quickly able to confirm that the Rajat's Challenger model does work more efficiently and accurately than the in-production Champion model. Over the next hour she progressively shifts customer traffic from Champion to Challenger and, once all traffic is going to the Champion, she undeploys Champion. At that point she triggers parallel deployments to the other regions and goes through the same A/B analysis in each to make sure that the behaviour is consistent across the galaxy where Weyland-Yutani's customers are found.
-
-Once the new Challenger model is taking all customer traffic in every galactic region, Annika retags the old Champion model with a version number, and tags the Challenger model as Champion:
-
-```sh
-$ ./kit tag 1.0 corp-registry/app-model:champion
-$ ./kit tag champion corp-registry/app-model:challenger
-```
+### Deployment
 
-Once the deployment is complete across all galactic regions, Annika sends an update to the AI/ML, app, and executive teams. Weyland-Yutani's leadership is ecstatic because the new model will give their spaceships a better way to scan for alien lifeforms when they receive random deep space distress signals. Everyone involved gets cake, champagne, and a high-five from Ripley herself...
+Adding the `challenger` tag to a ModelKit was a trigger in our team for the DevOps group to know that it was ready to deploy to production. They would take the serialized model from the ModelKit and ready it for deployment via our pipelines. For small infrequently changing models that would usually mean wrapping it in a container. For larger and more iterative models we typically use a Kubernetes `init` container linked to the serialized model file stored in a safe location in production (this makes iterations on the model faster, but requires more controls to ensure the model changes are properly tested).
```

Comment:
`stored in a safe location` almost suggests a 3rd party system but that is not the case it is stored on an available persistent volume. 

Also, init containers is not the only way to modify the pod with the model "post-start hooks", "sidecars" or even an "entrypoint" can be used. The main idea is to be able to modify the pod and bring the model. 

The size of the model is not really a sure way to decide on the method.